tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/TesseractOCRParser.java

```diff
@@ -239,25 +239,20 @@ private void parse(TikaInputStream tikaInputStream, File tmpOCROutputFile,
             if (size >= config.getMinFileSizeToOcr() && size <= config.getMaxFileSizeToOcr()) {
 
             	// Process image
-            	if (config.isEnableImageProcessing()) {
+            	if (config.isEnableImageProcessing() || config.isApplyRotation()) {
```

Comment:
I think this makes more sense than the consistency check.  If the user selected either, go for it.